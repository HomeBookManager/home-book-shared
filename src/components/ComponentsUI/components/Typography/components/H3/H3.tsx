import { FC } from 'react';

// components
import Typography from '../../Typography';

// types
import { TypographyFontType } from '../../enums';
import { TOmittedTypographyProps } from '../../types';

const H3: FC<TOmittedTypographyProps> = ({ children, ...restProps }) => (
  <Typography fontType={TypographyFontType.h3} {...restProps}>
    {children}
  </Typography>
);

export default H3;
