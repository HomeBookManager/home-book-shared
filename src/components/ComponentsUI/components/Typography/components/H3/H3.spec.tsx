import { render } from '@testing-library/react';

// components
import H3 from './H3';

describe('H3 snapshots', () => {
  it('should render H3', () => {
    //before
    const { asFragment } = render(<H3>H3</H3>);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
