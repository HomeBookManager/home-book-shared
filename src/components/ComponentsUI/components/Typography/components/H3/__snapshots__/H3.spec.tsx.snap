// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H3 snapshots should render H3 1`] = `
<DocumentFragment>
  <h3
    class="Typography Typography--regular Typography--normal Typography--h3"
    data-test-text=""
  >
    H3
  </h3>
</DocumentFragment>
`;
